### DISCUSSION TITLE
Apprise Notifications

### DISCUSSION THREAD

**Discussion by jacobabehm** [mcguirepr89/BirdNET-Pi#270] [Q&A]
Is something still incomplete with the apprise notifications?  

I have apprise configured to notify via email and SMS (Twilio).

When I toggle the switch for "Notify each new detection", I can see the change in birdnet.conf, I see the change update in the log, and I start getting notifications.

When I toggle the switch for "Notify each new species", I see none of the above (not even a line for it in birdnet.conf) and I never get those notifications.

**Comment by mcguirepr89** [comment]
Hi @jacobabehm 

There is a chance that it is required that changes to those settings require a service restart for the `birdnet_server.service` -- though the "cleanest" way to get that service restarted correctly is to use "Tools" > "Services" > "Restart ALL Services"

My guess would be that the script being used in your `birdnet_server.service` is still the old one that was loaded into memory when the service started.

Let me know if "Restart ALL Services" fixes things for you and we will add the service restart to the `config.php` when those settings are changed.

Thank for letting us know about the kink!

**Reply by ehpersonal38** [reply]
Actually I can confirm that neither notification option requires a restart. I did it that way because I was annoyed having to restart everytime I made a change in testing 😄

I'm going to do some investigation and will report back on this

**Comment by ehpersonal38** [comment]
Thank you for the bug report! I can confirm this issue is now fixed with d0e345051b498847c7c42abbefe3a20d6d61def5, so go ahead and update and let me know if that resolves things!

**Comment by oliverk71** [comment]
How is it used with Mastodon? 
I found the docs, but still don't have an idea how to actually use it: https://docs.joinmastodon.org/methods/apps/oauth/
